client/my-sites/email/email-management/home/email-plan.jsx

```diff
@@ -0,0 +1,319 @@
+			return null;
+		}
+
+		if ( ! selectedSite || ! purchase ) {
+			return <VerticalNavItem isPlaceholder />;
+		}
+
+		const managePurchaseUrl = getManagePurchaseUrlFor( selectedSite.slug, purchase.id );
+		return (
+			<VerticalNavItem path={ managePurchaseUrl }>
+				{ translate( 'Billing and payment settings' ) }
+			</VerticalNavItem>
+		);
+	}
+
+	getManageAllNavItemProps() {
+		const { currentRoute, domain, selectedSite } = this.props;
+
+		if ( hasGSuiteWithUs( domain ) ) {
+			return {
+				external: true,
+				path: getGoogleAdminUrl( domain.name ),
+			};
+		}
+
+		if ( hasTitanMailWithUs( domain ) ) {
+			if ( isEnabled( 'titan/iframe-control-panel' ) ) {
+				return {
+					path: emailManagementManageTitanAccount( selectedSite.slug, domain.name, currentRoute ),
+				};
+			}
+
+			return {
+				external: true,
+				path: emailManagementTitanControlPanelRedirect(
+					selectedSite.slug,
+					domain.name,
+					currentRoute
+				),
+			};
+		}
+
+		return {
+			path: emailManagementForwarding( selectedSite.slug, domain.name, currentRoute ),
+		};
+	}
+
+	renderManageAllNavItem() {
+		const { domain, translate } = this.props;
+
+		if ( ! domain ) {
+			return null;
+		}
+
+		if ( ! hasGSuiteWithUs( domain ) && ! hasTitanMailWithUs( domain ) ) {
+			return null;
+		}
+
+		const manageAllNavItemProps = this.getManageAllNavItemProps();
+
+		return (
+			<VerticalNavItem { ...manageAllNavItemProps }>
+				{ translate( 'Manage all mailboxes', {
+					comment:
+						'This is the text for a link to manage all email accounts/mailboxes for a subscription',
+				} ) }
+			</VerticalNavItem>
+		);
+	}
+
+	render() {
+		const {
+			domain,
+			selectedSite,
+			hasEmailPlanSubscription,
+			purchase,
+			isLoadingPurchase,
+			translate,
+		} = this.props;
+
+		const { statusClass, text, icon } = resolveEmailPlanStatus( domain );
+
+		const cardClasses = classnames( 'email-plan__general', statusClass );
+		const addMailboxProps = this.getAddMailboxProps();
+
+		return (
+			<>
+				{ selectedSite && hasEmailPlanSubscription && (
+					<QuerySitePurchases siteId={ selectedSite.ID } />
+				) }
+				<HeaderCake onClick={ this.handleBack }>{ this.getHeaderText() }</HeaderCake>
+				<CompactCard className={ cardClasses }>
+					<span className="email-plan__general-icon">
+						<EmailTypeIcon domain={ domain } />
+					</span>
+					<div>
+						<h2>{ domain.name }</h2>
+						<span className="email-plan__status">
+							<MaterialIcon icon={ icon } /> { text }
+						</span>
+					</div>
+				</CompactCard>
+
+				{ hasEmailPlanSubscription && (
+					<EmailPlanSubscription
+						purchase={ purchase }
+						domain={ domain }
+						selectedSite={ selectedSite }
+						isLoadingPurchase={ isLoadingPurchase }
+					/>
+				) }
+
+				<EmailPlanMailboxesList emails={ this.getEmails() } />
+
+				<div className="email-plan__actions">
+					<VerticalNav>
+						<VerticalNavItem { ...addMailboxProps }>
+							{ translate( 'Add new mailbox' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Add New Mailboxes' )` *ES Score: 8*